Ivan_Evseichik/Ansible start/04.Ansible.start.md

```diff
@@ -0,0 +1,169 @@
+ Ansible start
+-----------
+ - Install ansible in my Centos machine
+ ``` bash
+ sudo yum install epel-release && sudo yum install ansible
+ ```
+ - Create inventory with all variables and hierarchie
+ 	- change my hosts `/etc/ansible/hosts`
+ 	``` bash
+    [jump_srv]
+	178.124.206.48 ansible_user=jump
```

Comment:
Here also can be name

Reply:
I just did it in two versions, one through variables, and one with a simple description